Library/OcPciIoLib/OcPciIoLib.c

```diff
@@ -0,0 +1,76 @@
+/** @file
+   PciIo overrides
+
+   Copyright (c) 2023, xCuri0. All rights reserved.<BR>
+   SPDX-License-Identifier: BSD-2-Clause-Patent
+
+ **/
+
+#include "OcPciIoU.h"
+#include <Library/OcPciIoLib.h>
+
+EFI_CPU_IO2_PROTOCOL *
+OcPciIoInstallProtocol (
+  IN BOOLEAN  Reinstall
+  )
+{
+  EFI_TPL                          tpl;
+  UINTN                            Index;
+  EFI_STATUS                       Status  = 0;
```

Comment:
0 is not an EFI_STATUS value, also declaration and initialisation must be separate.